### ISSUE TITLE
SessionID checks

### ISSUE THREAD

**Issue by altanai** [AmpleChat/brightchats#7]
Redirect to contacts page if session id is absent or wrong on api validation

**Comment by altanai** [comment]
added checks and redirection userid and email . Session ID check not incorporated yet 
cc @FarookAfsari

**Comment by FarookAfsari** [comment]
Agreed, let's add session id to allow to proceed; if not then redirect to the contacts page.  BUT, need to test that it does not create new issues.
Farook.
